schemas/twitter.js

```diff
@@ -0,0 +1,167 @@
+import * as twitter from '../apis/twitter';
+
+import {
+    GraphQLSchema,
+    GraphQLObjectType,
+    GraphQLID,
+    GraphQLString,
+    GraphQLNonNull,
+    GraphQLInt,
+    GraphQLList,
+} from 'graphql';
+
+let UserType = new GraphQLObjectType({
+
+    name: 'TwitterUser',
+    description: 'Twitter user, or as we like to say in France: a \'tweetos\'',
+    fields: () => ({
+        created_at: { type: GraphQLString },
```

Comment:
nit can we align consecutive colons here? it's the [style in other schemas](https://github.com/clayallsopp/graphqlhub/blob/master/schemas/hn.js#L257)

filed an issue to integrate ESLint so the tests ask for this automatically https://github.com/clayallsopp/graphqlhub/issues/5